pykechain/client.py

```diff
@@ -1378,6 +1378,60 @@ def create_proxy_model(
         )
         return self._create_part(action='create_proxy_model', data=data, **kwargs)
 
+    def _create_parts_bulk(
+            self,
+            parts: List[Dict],
+            **kwargs
+    ) -> PartSet:
+        """
+        Create multiple part instances simultaneously.
+
+        :param parts: list of dicts, each specifying a part instance. Available fields per dict:
+            * name: str
+            * parent_id: str of parent instance
+            * model_id: str of part model
+            * properties: list of dicts, each specifying a property to update. Available fields per dict:
```

Comment:
refer to documentation on 'create part with properties' -- should be the same interface.

Reply:
added, please re-review